webapp/components/admin_console/signup_settings.jsx

```diff
@@ -51,7 +51,7 @@ export default class SignupSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.email.requireVerificationTitle'
-                            defaultMessage='Require Email Verification: '
+                            defaultMessage='Enable Email Verification: '
```

Comment:
This seems more like "Require" instead of "Enable" since it's not optional after it's turned on.